workflows/bacterial_genomics/quality_and_contamination_control/quality_and_contamination_control-tests.yml

```diff
@@ -0,0 +1,63 @@
+- doc: Test outline for quality_and_contamination_control.ga
+  job:
+    Input sequence reads (forward):
+      class: File
+      path: https://zenodo.org/records/11484215/files/paired_r1.fastq.gz
+      filetype: fastq.gz
+    Input sequence reads (reverse):
+      class: File
+      path: https://zenodo.org/records/11484215/files/paired_r2.fastq.gz
+      filetype: fastq.gz
```

Comment:
```suggestion
      filetype: fastqsanger.gz
```

Reply:
It's not that important, but many (other) tools only accept inputs in sanger encoding, and everything is sanger encoded these days.